installer/templates/phx_web/views/helpers.ex

```diff
@@ -1,10 +1,79 @@
-defmodule <%= @web_namespace %>.ErrorHelpers do
+defmodule <%= @web_namespace %>.Helpers do
   @moduledoc """
   Conveniences for translating and building error messages.
   """<%= if @html do %>
 
   use Phoenix.HTML
 
+  import Phoenix.LiveView
+  import Phoenix.LiveView.Helpers
+
+  alias Phoenix.LiveView.JS
+
+  @doc """
+  Renders a flash message.
+
+  The rendered flash receives a `:type` that will be used to define
+  proper classes to the element, and a `:message` which will be the
+  inner HTML, if any exists.
+
+  ## Examples
+
+      <.flash type="info" message="User created" />
+  """
+  def flash(assigns) do
+    if message = Map.get(assigns.flash, assigns.kind) do
+      ~H"""
+      <p class={"alert alert-#{@type}"} role="alert" phx-click="lv:clear-flash" phx-value-key={@kind}>
+       <%= @message %>
+      </p>
+      """
+    end
+  end
+
+  @doc """
+  Renders a live component inside a modal.
+
+  The rendered modal receives a `:return_to` option to properly update
+  the URL when the modal is closed.
+
+
+  """
+  def modal(assigns) do
```

Comment:
We also have `live_helpers`: https://github.com/phoenixframework/phoenix/blob/master/priv/templates/phx.gen.live/live_helpers.ex#L1

I guess we should either move these functions to there or remove this that file if we want to keep it here 🤔